### PR TITLE
Fixes to issues identified by CodeQL Scanning

### PR DESCRIPTION
**Pull request by RNKuhns**
#### What does this implement/fix? Explain your changes.

 This includes 2 fixes to errors that were raised by #68. Fixing this will allow #68 to be merged.

Fixes call init on parent (BaseObject) in test classes. Also, updates conf.py to fix minor issues with unused variables.

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -65,9 +66,9 @@
     extensions += ["sphinx_sitemap"]
 
     # -- Sitemap -------------------------------------------------------------
-    html_baseurl = os.environ.get("SITEMAP_URL_BASE", "http://127.0.0.1:8000/")
```
Comment (fkiraly) [inline docs/source/conf.py:68]: what was this??

File: skbase/tests/test_base.py [hunk 2]
```diff
@@ -177,6 +177,7 @@ def __init__(self, foo, bar=84):
         self.foo = foo
         self.foo_ = deepcopy(foo)
         self.bar = bar
+        super().__init__()
```
Comment (fkiraly) [inline skbase/tests/test_base.py:180]: I think we should test for that this is caught.

In `sktime` it is caught by checking whether `_tags_dynamic` is initialized.

**Review by fkiraly — APPROVED** [review]
looks ok, assuming the mysterious URL related changes are unimportant.

Perhaps you can remove the commented code too?

File: docs/source/conf.py (hunk 1 above)
Comment (RNKuhns) [reply docs/source/conf.py:68]: @fkiraly it got copied from elsewhere.

File: skbase/tests/test_base.py (hunk 2 above)
Comment (RNKuhns) [reply skbase/tests/test_base.py:180]: I think that makes sense. Will verify whether we have the test and if not add it as part of #62.

**Comment by RNKuhns** [comment]
> looks ok, assuming the mysterious URL related changes are unimportant.
> 
> Perhaps you can remove the commented code too?

Commented out code removed, since that part wasn't needed.

**Pull request merged**